server/config.ts

```diff
@@ -71,4 +71,7 @@ export default {
     },
   },
   domain: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
+  slack: {
+    workspaceId: get('SLACK_WORKSPACE_ID', 'T02DYEB3A', requiredInProduction),
```

Comment:
So this wants to be set in `values-*.yaml`, the value here is the default if not set. If it's not going to change across envs then it should be set in `values.yaml` 👍